app/components/views/Bond/index.tsx

```diff
@@ -749,10 +754,39 @@ export const Bond: FC<Props> = (props) => {
                 <Spinner />
               </div>
             ) : (
-              <ButtonPrimary
-                {...getButtonProps()}
-                className={styles.submitButton}
-              />
+              <Box
+                sx={{ display: "flex", flexDirection: "column", width: "100%" }}
+              >
+                <ButtonPrimary
+                  {...getButtonProps()}
+                  className={styles.submitButton}
+                />
+                {view === "redeem" && !showSpinner && (
+                  <Box
+                    sx={{
+                      display: "flex",
+                      width: "100%",
+                      alignItems: "center",
+                      justifyContent: "center",
+                      paddingRight: "16px",
+                    }}
+                  >
```

Comment:
Don't think we need `Box`, prefer plain divs and plain css in `styles.ts`.

I like to keep styles out of JSX/js files as best as I can :)

Reply:
updated, I will keep this in mind for the future.